.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         go install github.com/mattn/goveralls@latest
     - if: ${{ runner.os == 'Linux' }}
       run: if [ $(goimports -l .) ]; then goimports -d .; echo 'Failed the goimports format check. Please format the code using "goimports -w ."'; exit 1; fi
-    - run: go test -mod=mod -coverpkg="./..." -coverprofile=covprofile ./...
+    - run: go test $(go list ./... | grep -v /transformation_test) -mod=mod -coverpkg="./..." -coverprofile=covprofile ./...
```

Comment:
The test should skip itself (with `t.Skip`); a bare `go test` should still work.